datafusion/src/logical_plan/builder.rs

```diff
@@ -289,6 +289,14 @@ impl LogicalPlanBuilder {
         }))
     }
 
+    pub fn window(
+        &self,
+        group_expr: impl IntoIterator<Item = Expr>,
+        aggr_expr: impl IntoIterator<Item = Expr>,
```

Comment:
would eventually need sort_exprs too to handle the various window clauses

Maybe worth taking a peek at how the `sqlparser` crate structures things so the API on the builder is "future proof" (aka we don't have to break backwards compatibility when we add support for non empty over clause)

https://github.com/ballista-compute/sqlparser-rs/blob/35ef0eee3857c59827b2587bc813218a00dce264/src/ast/mod.rs#L372-L376